examples/with-nextjs-and-clerk-auth/src/lib/clerk-api/types.d.ts

```diff
@@ -46,3 +46,11 @@ export type OrganizationDomainWebhookEvent =
         deleted: boolean;
       }
     >;
+
+export type ThemeVariant = 'monite' | 'material';
+export type ThemeMode = 'light' | 'dark';
+export type SelectedTheme = [theme: ThemeVariant, mode: ThemeMode];
```

Comment:
From Clerk's point of view, he knows nothing about `ThemeVariant | ThemeMode | SelectedTheme` - all of these types are part of the Next.js implementation of the application. It doesn't seem to me that this file is the best place for these types.